fapolicy_analyzer/ui/main_window.py

```diff
@@ -275,3 +304,25 @@ def on_analyzeMenu_activate(self, menuitem, *args):
     def on_trustDbMenu_activate(self, menuitem, *args):
         self.__pack_main_content(router(ANALYZER_SELECTION.TRUST_DATABASE_ADMIN))
         self.__set_trustDbMenu_sensitive(False)
+
+    def on_fapaPreferencesMenu_activate(self, menuitem, data=None):
+        logging.debug("on_fapaPreferencesMenu_activate() invoked.")
+
+    def on_fapdPreferencesMenu_activate(self, menuitem, data=None):
+        logging.debug("on_fapdPreferencesMenu_activate() invoked.")
```

Comment:
Are these tied to something?

Reply:
No. I pulled the menu items although I was originally considering having a dialog to set the daemon in permissive mode for instance, or configuring the analyzer's color settings. As menu items were removed from the original prototype, removing associated callbacks were in the plans before moving off of Draft status. The priority was/is Redux.

Reply:
Aspirational code removed.